gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -837,17 +836,17 @@ def _assert_variant_lines_equal(self, actual, expected):
     self.assertEqual(len(actual_fields), len(expected_fields))
     self.assertEqual(actual_fields[0], expected_fields[0])
     self.assertEqual(actual_fields[1], expected_fields[1])
-    self.assertItemsEqual(actual_fields[2].split(';'),
+    self.assertCountEqual(actual_fields[2].split(';'),
```

Comment:
Again, please confirm these asserts are as strong as the old ones.

Reply:
Yeah, they are identical - https://six.readthedocs.io/#six.assertCountEqual